mesonbuild/wrap/wrap.py

```diff
@@ -228,6 +232,26 @@ def get_hg(self, p):
                 subprocess.check_call(['hg', 'checkout', revno],
                                       cwd=checkoutdir)
 
+    def get_svn(self, p):
+        checkoutdir = os.path.join(self.subdir_root, p.get('directory'))
+        revno = p.get('revision')
+        is_there = os.path.isdir(checkoutdir)
+        if is_there:
+            current_revno = subprocess.getoutput(' '.join(['svn', 'info', '--show-item', 'revision', checkoutdir]))
```

Comment:
Use `Popen_safe` from `mesonlib` instead. It does a lot of portability magic.

Reply:
To clarify, the portability magic is required because you are parsing the output of the command. The other instances of `subprocess` invocation are fine.